sky/client/sdk.py

```diff
@@ -1819,8 +1819,8 @@ def api_login(endpoint: Optional[str] = None) -> None:
             config_path.touch()
```

Comment:
Do we need to change line 1818 as well, in case of no user config file existing but project config existing?